ex3-solution/src/components/meta/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import "./meta.css";
+
+const Meta = ({ cart = true, account = true, search = true }) => (
```

Comment:
i'm pretty sure it should be `{ cart = true, account = true, search = true } = {}`